client/blocks/reader-full-post/index.jsx

```diff
@@ -286,41 +286,21 @@ export class FullPostView extends Component {
 			: undefined;
 
 	// Scroll to the top of the comments section.
-	scrollToComments = () => {
-		if ( ! this.props.post ) {
-			return;
-		}
-		if ( this.props.post._state ) {
-			return;
-		}
-		if ( this._scrolling ) {
+	scrollToComments = ( { focusTextArea = false } = {} ) => {
+		if ( ! this.props.post || this.props.post._state || this._scrolling ) {
 			return;
 		}
 
 		this._scrolling = true;
-		setTimeout( () => {
-			const commentsNode = this.commentsWrapper.current;
-			if ( commentsNode && commentsNode.offsetTop ) {
-				scrollTo( {
-					x: 0,
-					container: this.readerMainWrapper.current,
-					y: commentsNode.offsetTop - 48,
-					duration: 300,
-					onComplete: () => {
-						// check to see if the comment node moved while we were scrolling
-						// and scroll to the end position
-						const commentsNodeAfterScroll = this.commentsWrapper.current;
-						if ( commentsNodeAfterScroll && commentsNodeAfterScroll.offsetTop ) {
-							window.scrollTo( 0, commentsNodeAfterScroll.offsetTop - 48 );
-						}
-						this._scrolling = false;
-					},
-				} );
+		scrollToComments( {
```

Comment:
I moved this to a new file client/blocks/reader-full-post/scroll-to-comments.tsx so I could reuse the logic inside the EngagmentBar component [here](https://github.com/Automattic/wp-calypso/pull/96172/files#diff-dcc9916e9c34067da838dee64179aaf8aca0c92026d439fc097b02f928834173R28).